spring-cloud-gcp-docs/pom.xml

```diff
@@ -90,6 +90,7 @@
 								<snippets>${basedir}/target/generated-snippets</snippets>
 								<github-tag>${github-tag}</github-tag>
 								<docinfo>true</docinfo>
+								<spring-integration-version>4.3.12.RELEASE</spring-integration-version>
```

Comment:
`4.3.13` is the latest in that line. 
Although there is already `5.0.0`: https://projects.spring.io/spring-integration

However I don't understand why do we need this explicit value here since we might be able to pick up it from the parent Spring Boot. Something like:
```
<spring-integration-version>${spring-integration.version}</spring-integration-version>
```

Reply:
Yeah I was thinking the same, but `${spring-integration.version}` isn't recognised in the `spring-cloud-gcp-docs` `pom.xml`.
I tried adding spring-boot's BOM but it also doesn't work...